rpcd/playbooks/roles/rpc_support/tasks/create_default_flavors.yml

```diff
@@ -0,0 +1,43 @@
+---
+# Copyright 2017, Rackspace US, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+- name: Install shade and its dependencies
+  pip:
+    name: "{{ item }}"
+    state: latest
+    extra_args: "--isolated"
```

Comment:
It fails if we do not  use it because of the version of shade dependencies are a different version than what the repo has.